gnosis/safe-cmm-app-react
# Gnosis protocol price source

# Description

## :warning: depends on #145, review/merge this only after that PR is merged

Added Gnosis Protocol as new price source

Using the new price source on Market price calculation of Bracket visualization on strategy page

# To Test
1. Make sure there's an active strategy
1. Go to strategies page
1. Expand the drop down of any active strategy page

- [ ] Market price should be displayed and reflect GP prices

## Note: easier to notice the difference on `xDai`, as `1inch` usually does not provide prices for it.

# Background

Considering using GP prices everywhere for `xDai`. Not on this PR though.
